stable/wordpress/templates/secrets.yaml

```diff
@@ -14,9 +14,4 @@ data:
   {{ else }}
   wordpress-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
-  {{ if .Values.databasePassword }}
```

Comment:
Indeed. This secret is not used anywhere, the password is pulled from the MariaDB subchart. Having a password secret around that is not used is just confusing to the user imo.